### PR TITLE
Fixes README minimal example and send string values to `core.setOutput`

### LINKED ISSUE #9: Minimal example from README doens't work
When I tried to use the following workflow
```yml
name: Minimal example   ## fixed 'Name' typo 
on: [push]

jobs:
  minimal_example:
    runs-on: ubuntu-latest
    steps:
      - id: action_with_json_output
        run: 'echo ::set-output name=data::{ "foo": { "bar":"baz" } }'
      - id: data
        uses: gr2m/get-json-paths-action@v1.x
        with:
          json: ${{ steps.action_with_json_output.outputs.data }}
          bar: "foo.bar"
      - run: "echo bar is ${{ steps.data.outputs.bar }}"
```
I got the error `SyntaxError: Unexpected token f in JSON at position 2` because the `steps.action_with_json_output.outputs.data` value is the string `'{ foo: { bar:baz } }'`
output

---

To fix this I just change the `data` output to `{ \"foo\": { \"bar\":\"baz\" } }`.

Actually I think in the line below `value` should be stringified because the second param of `setOutput` is expected to be a string
https://github.com/gr2m/get-json-paths-action/blob/1ded8f388247a1d62f74718206877cd75d961ac9/index.js#L14-L16

### PR DESCRIPTION
**Pull request by micalevisk**
Closes #9

Before ba9ed1bf108b8a09edecd8d9a991a4ede654e986:
workflow before

After:
workflow after

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -67,8 +67,9 @@ try {
 
   for (const [name, path] of Object.entries(paths)) {
     const value = get(jsonParsed, path);
-    core.debug(`setting output ${name} to ${value} using "${path}"`);
-    core.setOutput(name, value);
+    const strValue = JSON.stringify(value);
```
Comment (gr2m) [inline dist/index.js:70]: Perfect 👍

**Review by gr2m — APPROVED** [review]
This looks great, thanks a lot!

**Comment by juliangruber** [comment]
this was a breaking change for me, as I'm using this action's output in string interpolation

**Comment by micalevisk** [comment]
> this was a breaking change for me, as I'm using this action's output in string interpolation

sorry I missed that.

So @gr2m needs to remove the release v1.0.3 and releases v2? because `core.setOutput` method expects the second parameter to be a string: 
https://github.com/actions/toolkit/blob/5859d7172e5f4281198f6d74eeb9fe96fb27a23d/packages/core/src/core.ts#L83
Maybe change the action's description to **serialized values** as well
https://github.com/gr2m/get-json-paths-action/blob/c4ee500e4f251c03d878f2be8d0f6ca233c6a0cb/action.yml#L2
idk if I'm right tho.

**Comment by gr2m** [comment]
Very sorry @juliangruber, I'll revert the last release for now. Can you point me to your usage that broke now, for reference? I want to add a test to prevent that from happening again.

**Comment by juliangruber** [comment]
No worries, thanks to you publishing tags as well I could simply downgrade the version.

I'm using the action to get the `package.json`'s '.version', from which I construct asset paths.

https://github.com/hypergraph-xyz/desktop/blob/18f0e61fc2aff079bc8f21d768b058827041a9c8/.github/workflows/ci.yml#L47-L58

```yml
      - name: get version
        id: version
        uses: gr2m/get-json-paths-action@v1.0.2
        with:
          json: ${{ steps.package.outputs.content }}
          version: 'version'
      - name: archive mac dmg
        if: startsWith(matrix.os, 'macos')
        uses: actions/upload-artifact@v1
        with:
          name: mac
          path: dist/Hypergraph-${{ steps.version.outputs.version }}.dmg
```

**Comment by gr2m** [comment]
The new version is already out: https://github.com/gr2m/get-json-paths-action/releases/tag/v1.0.4

Thanks Julian, I'll create a test first, then take it from there.

@micalevisk Please give me a moment, I'll probably get to this early next week

**Comment by micalevisk** [comment]
@gr2m just be aware that with the first screenshot code, [v1.2.3 of @actions/core](https://github.com/actions/toolkit/tree/core-1.2.3/packages/core) will break this action due to the following error

```
/github/workspace/.github/actions/get-json-paths-action/node_modules/@actions/core/lib/command.js:66
        .replace(/%/g, '%25')

TypeError: (s || "").replace is not a function
    at escapeData (/github/workspace/.github/actions/get-json-paths-action/node_modules/@actions/core/lib/command.js:66:10)
    at Command.toString (/github/workspace/.github/actions/get-json-paths-action/node_modules/@actions/core/lib/command.js:60:35)
    at issueCommand (/github/workspace/.github/actions/get-json-paths-action/node_modules/@actions/core/lib/command.js:23:30)
    at Object.issue (/github/workspace/.github/actions/get-json-paths-action/node_modules/@actions/core/lib/command.js:27:5)
    at error (/github/workspace/.github/actions/get-json-paths-action/node_modules/@actions/core/lib/core.js:127:15)
    at Object.setFailed (/github/workspace/.github/actions/get-json-paths-action/node_modules/@actions/core/lib/core.js:101:5)
    at Object.<anonymous> (/github/workspace/.github/actions/get-json-paths-action/index.js:19:8)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
```

**Comment by gr2m** [comment]
Thanks @micalevisk. I'm currently taking care of family, I hope to be back to work in a few days

**Pull request merged**